File: cpp/263_丑数_easy.cpp
class Solution {
public:
    bool isUgly(int n) {
        if (n == 0) return false;
        int a[3] = {2, 3, 5};
        for (auto num : a) 
            while (n % num == 0)
                n /= num;
        return n == 1 ? true : false;
    }
};

File: python/263_丑数_easy.py
class Solution:
    def isUgly(self, n: int) -> bool:
        if not n:
            return False
        a = [2, 3, 5]
        for _a in a:
            while n % _a == 0:
                n //= _a
        return True if n == 1 else False